qemu-psa-sp.mk

```diff
@@ -0,0 +1,141 @@
+OUT_PATH		?= $(ROOT)/out
+
+QEMU_VIRTFS_HOST_DIR	?= $(ROOT)
+QEMU_VIRTFS_AUTOMOUNT	?= y
+
+QEMU_USE_CUSTOM_DTB		?= y
+
+MEASURED_BOOT			?= y
+MEASURED_BOOT_FTPM		?= n
+TS_SMM_GATEWAY			?= y
+TS_UEFI_TESTS			?= y
+# Supported values: embedded (fip is currently not supported on QEMU platform)
+SP_PACKAGING_METHOD		?= embedded
+SPMC_TESTS			?= n
+
+# Disable packages not used by this configuration
+BR2_PACKAGE_HOST_E2FSPROGS	?= n
+BR2_PACKAGE_KEYUTILS		?= n
+BR2_PACKAGE_MMC_UTILS		?= n
+BR2_PACKAGE_OPENSC		?= n
+BR2_PACKAGE_OPTEE_EXAMPLES_EXT	?= n
+BR2_PACKAGE_STRACE		?= n
+
+# Building xtest is not necessary if we don't want to run the SPMC tests
+ifneq ($(SPMC_TESTS),y)
+BR2_PACKAGE_OPTEE_TEST_EXT	?= n
+BR2_PACKAGE_LIBOPENSSL		?= n
+BR2_PACKAGE_OPENSSL		?= n
+endif
+
+# TS SP configurations
+DEFAULT_SP_CONFIG		?= default-opteesp
+SP_BLOCK_STORAGE_CONFIG		?= $(DEFAULT_SP_CONFIG)
+SP_PSA_ITS_CONFIG		?= $(DEFAULT_SP_CONFIG)
+SP_PSA_PS_CONFIG		?= $(DEFAULT_SP_CONFIG)
+SP_PSA_CRYPTO_CONFIG		?= $(DEFAULT_SP_CONFIG)
+SP_PSA_ATTESTATION_CONFIG	?= $(DEFAULT_SP_CONFIG)
+SP_SMM_GATEWAY_CONFIG		?= $(DEFAULT_SP_CONFIG)
+
+# SPMC runs at EL1
+SPMC_AT_EL ?= 1
+# Use GICv2 or GICv3
+GICV3 ?= y
+# Use TF-A's Trusted Board Boot mechanism?
+TF_A_TRUSTED_BOARD_BOOT ?= y
```

Comment:
This switch seems unused. Is there even an altternate to TF-A?

Reply:
It is used in `qemu_v8.mk`, I am ot sure what you are referring to.

Reply:
Right, sorry. discard my comment.